tests/playwright/Report.md

```diff
@@ -0,0 +1,124 @@
+
+### Project Context
+The Playwright evaluation is a result of a need for potential replacement of [WebdriverIO](https://webdriver.io/) with one of the best non-selenium modern automation testing framwork. Wikimedia supports a large number of repositories as well as extensions. In order to ensure good code practices across all these repositories, an extensive amount of testing is performed and one of the tests performed is an end to end test. WebdriverIO is the current browser automation framework being used for implementing the end-to-end tests. However, with the recent increase in awareness about end-to-end testing, a number of equally competitive non-selenium solutions have been introduced and one of them is [Microsoft Playwright](https://github.com/microsoft/playwright).
+
+At the time of choosing which automation framework to use in the implementation of the automation testing, extensive research was done and an [evaluation of WebdriverIO](https://filipin.eu/selenium-javascript) was carried out by Željko Filipin, who is a Senior Engineer in Test at Wikimedia Foundations and also one of my mentors. In search for a potential replacement of WebdriverIO, a number of modern and non-selenium automation frameworks have been evaluated against WebdriverIO. These evaluations were done by Soham who was Google Summer of Code 2020 intern and one of my mentors, they can be found using the links below;
+- [Cypress](https://www.sohamp.dev/blog/2020-06-29-fanboying-cypress/)- to be confirmed and changed
+- [Puppeteer](https://www.sohamp.dev/blog/2020-08-24-wmf-gsoc2020/)
+
+
+So, this brings us to the current framework which is [Microsoft Playwright](https://github.com/microsoft/playwright) being evaluated against WebdriverIO. The project involved;
+- Configuring and Setting up of Wikimedia-core to run on CI, in this case on Github actions.
+- Implementation of all Wikimedia-core tests in Playwright
+
+### Playwright Vs WebdriverIO
+
+[WebdriverIO](https://github.com/webdriverio/webdriverio) WebdriverIO is a test automation framework that allows you to run tests based on the Webdriver protocol and Appium automation technology. WebdriverIO is known to be;
+Extendable
+Adding helper functions, or more complicated sets and combinations of existing commands is simple and really useful
+
+Compatible
+WebdriverIO can be run on the WebDriver Protocol for true cross-browser testing as well as Chrome DevTools Protocol for Chromium-based automation using Puppeteer.
+
+Feature Rich
+The huge variety of community plugins allows you to easily integrate and extend your setup to fulfill your requirements.
+
+The above points are just a few of WebdriverIO capability. To explore more WebdriverIO capability, you can check out the [WebdriverIO](https://webdriver.io/) documentation and the [Wikipedia Repo](https://github.com/wikimedia/mediawiki/tree/master/tests/selenium).
```

Comment:
Instead of linking to https://github.com/wikimedia/mediawiki/tree/master/tests/selenium, I would link to https://www.mediawiki.org/wiki/Selenium

Reply:
Noted. Thanks